pkg/controllers/cluster/cluster_controller.go

```diff
@@ -121,7 +121,7 @@ func (cc *ClusterReconciler) Reconcile(request ctrl.Request) (ctrl.Result, error
 	if !reflect.DeepEqual(c.Status, copy.Status) {
 		logger.Info(ctx, "Writing cluster status.")
 		if err := cc.Status().Update(ctx, copy); err != nil {
-			logger.Error(ctx, "Failed to update cluster status", "error", err)
+			logger.Info(ctx, "Failed to update cluster status", "error", err)
```

Comment:
We want to log only conflicts on Info level. Other errors should be logged using Error.
You can use apierrors.IsConflict(err).

Reply:
fixed